packages/commonwell-sdk/src/models/address.ts

```diff
@@ -40,3 +43,65 @@ export function normalizeStatePreprocess(arg: unknown): unknown {
   if (typeof arg === "string") return normalizeState(arg);
   return undefined;
 }
+
+export function normalizeStatePreprocessSafe(arg: unknown): unknown {
+  if (typeof arg === "string" && ["", "undefined", "null"].includes(arg.trim())) return undefined;
+  if (typeof arg === "string") return normalizeStateSafe(arg);
+  return undefined;
+}
+
+/**
+ * Validates if an address line has sufficient content
+ */
+function isValidAddressLine(line: string[] | null | undefined): boolean {
+  if (!line || line.length === 0) return false;
+  return line.some(lineItem => lineItem.trim().length >= MIN_LINE_LENGTH);
```

Comment:
should we filter out the ones with not enough chars? thinking that if we leave the first one w/ 1 char, for example, that's the one we'll use in some places?

Reply:
Done on the SDK below